fastlane/lib/fastlane/actions/download_dsyms.rb

```diff
@@ -34,6 +34,14 @@ def self.run(params)
         end
       end
 
+      def self.download(url)
+        uri = URI.parse(url)
+        http = Net::HTTP.new(uri.host, uri.port)
+        http.use_ssl = (uri.scheme == "https")
+        res = http.get(uri.request_uri)
+        res.body
+      end
```

Comment:
Good idea. Doing a quick search over the rest of our codebase, it feels like we're very inconsistent in the application of this strategy.

As a note for us for the future, this feels like something we should work into `fastlane_core` and use everywhere else.